core/fraggle-rock/config/filters.js

```diff
@@ -31,7 +31,13 @@ const filterResistantAuditIds = ['full-page-screenshot'];
 
 // Some artifacts are used by the report for additional information.
 // Always run these artifacts even if audits do not request them.
-const filterResistantArtifactIds = ['HostUserAgent', 'HostFormFactor', 'Stacks', 'GatherContext'];
+const filterResistantArtifactIds = [
```

Comment:
Is this a result of the fact that `BaseArtifacts` and `FRBaseArtifacts` are different?

can we align this thing to the types to ensure there arent others slipping through the cracks?

Reply:
> Is this a result of the fact that BaseArtifacts and FRBaseArtifacts are different?

There is this:
https://github.com/GoogleChrome/lighthouse/blob/d43485bae2b86fcc940d27063f1677087a4f69a7/types/artifacts.d.ts#L74-L90
But that list includes artifacts that we *don't* want to be filter resistant (e.g. `InstallabilityErrors`)

I think this list should include artifacts that are not run in all 3 modes, but should always be run if they can. For example, `NetworkUserAgent` isn't available in snapshot mode.

Also, looks like `HostUserAgent`, `HostFormFactor` and `GatherContext` don't need to be filter resistant because they are base artifacts.